src/command_func.h

```diff
@@ -27,6 +27,11 @@
  */
 static const CommandCost CMD_ERROR = CommandCost(INVALID_STRING_ID);
 
+/**
+ * Define a return value for a failed command with no cost and no error.
+ */
+static const CommandCost CMD_NOTHING = CommandCost();
```

Comment:
`CommandCost()` is currently "command succeeded, but cost nothing". See for example `CmdPause`, `CmdForceTrainProceed`, or `CmdStartStopVehicle`.
I'm not sure documenting it as a a failure that is not an error and costs nothing is really warranted, or clear.

Reply:
Ah, thanks. I guess this does touch AIs then (not that it necessarily breaks anything, but needs considering).

Reply:
Also, there are plenty of commands returning `CommandCost()` already, adding another way to do the same thing seems like a bad idea.

Reply:
Good point, thanks.